### ISSUE TITLE
1 issue and 1 iput

### ISSUE THREAD

**Issue by donbene** [zniezelski/Printerceptor#2]
Hi,
First of all...your script is awesome and sooo helpful...thanks very much!

We have implemented this script in our windows environment and found one issue which I like to report to you:

In "printers.ps1" script on line 789 should be "-or" and not "-and" (see below:)

Your Script:
# Don't  set the default if printer isn't to be renamed or set to be recreated

```
                   if (($NoRename -eq $true) -and ($Recreate -eq $false)) { } else 
```

Changed Script:
# Don't  set the default if printer isn't to be renamed or set to be recreated

```
                   if (($NoRename -eq $true) -or ($Recreate -eq $false)) { } else 
```

The wrong default Printer was set. With the change above everything is working.

Sooo and here is the input for the multilingual complement (only Swiss-German) in your script:

if ((get-culture).name -eq "de-CH"){set-ItemProperty -Path $key -Name "RedirectedExpression" -Value "( (umgeleitet (\d+)))"  -Force}

Many thanks!!!
Kind regards donbene

**Comment by zniezelski** [comment]
Donbene,

Thanks for the feedback and troubleshooting work, it means a lot to me. I was able to recreate the problem by testing with excluding the default printer from rename/recreate. I created a Version 2.6 release that should fix this problem and added entries for the de-CH culture.

I changed a little bit more than stated above, please be sure to download the new release from github. Let me know if you experience any more trouble or have a feature recommendation.

I hope to have another release soon that allows group/user scope restriction and doesn't start doing anything to printers until the first configuration save is made.

With much appreciation,

Zachary Niezelski

**Comment by zniezelski** [comment]
Please use the 3.0 Release. I published the wrong de-CH expression in 2.6.

Thanks,

Zach
